src/pages/patch/patchTabs/Tasks.tsx

```diff
@@ -46,45 +42,30 @@ export const Tasks: React.FC<Props> = ({ taskCount }) => {
 
   const { search } = useLocation();
 
-  const [initialQueryVariables] = useState(
-    getQueryVariables(search, resourceId)
-  );
+  const queryVariables = getQueryVariables(search, resourceId);
 
-  const { sortBy, sortDir } = initialQueryVariables;
+  const { sortBy, sortDir, limit, page } = queryVariables;
 
   const columns = useSetColumnDefaultSortOrder<TaskResult>(
     columnsTemplate,
     sortBy,
     sortDir
   );
 
-  const { data, error, networkStatus, refetch } = useQuery<
+  const { data, error, loading, startPolling, stopPolling } = useQuery<
     PatchTasksQuery,
     PatchTasksQueryVariables
   >(GET_PATCH_TASKS, {
-    variables: initialQueryVariables as PatchTasksQueryVariables,
-    notifyOnNetworkStatusChange: true,
-    fetchPolicy: "network-only",
```

Comment:
we should change this to `cache-and-network` since the default fetchPolicy is `cache-first`